server/src/integrations/message-queue/drivers/pg-boss.driver.ts

```diff
@@ -0,0 +1,27 @@
+import PgBoss from 'pg-boss';
```

Comment:
Could we please create a file in docs/contributor/server/queue.mdx with very basic context on how queues work?

Reply:
sure, i will add